# KondoServoFeedbackMsg.msg
uint8 part_id
uint8 id
uint16 feedback_degree
uint8 current
uint8 temp
bool is_freed
# see the definitions in KondoServo.h
